### PR TITLE
github: fix cleanup to always run

### PR DESCRIPTION
**Pull request by michalskrivanek**
Fixes issue # (delete if not relevant)

## Changes introduced with this PR

*

*

*

## Are you the owner of the code you are sending in, or do you have permission of the owner?

[y/n]

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -26,6 +26,7 @@ jobs:
     - name: make
       run: |
         cd engine-appliance
+        exit 1
```
Comment (lveyde) [inline .github/workflows/build.yml:29]: This is just for testing?

Comment (michalskrivanek) [reply .github/workflows/build.yml:29]: yes:)

**Comment by michalskrivanek** [comment]
now working ok
when "keep_dir" requested the cleanup step "fails" so it's clear it's not cleared up

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -55,7 +55,7 @@ jobs:
     - name: Clean up entire workdir
       if: always()
       env:
-        KEEP_WORKDIR: ${{ inputs.keep_workdir }}
+        KEEP_WORKDIR: ${{ github.event.inputs.keep_workdir }}
       run: |
         [[ -z "${KEEP_WORKDIR}" ]] && rm -rf $PWD/*
```
[the remark this answers is not in the document]
Comment (michalskrivanek) [reply .github/workflows/build.yml:59]: it probably would

**Review by lveyde — APPROVED** [review]

**Review by sandrobonazzola — APPROVED** [review]

**Pull request merged**